docs/index.md

```diff
@@ -1,19 +1,25 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: PowerCards
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
Can change this to 
`[![CI Status](https://github.com/AY2223S2-CS2103T-W11-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2223S2-CS2103T-W11-3/tp/actions)`

Reply:
yepp sure